staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -551,14 +543,22 @@ type SimpleProxyReactor struct {
 }
 
 func (r *SimpleProxyReactor) Handles(action Action) bool {
-	resourceCovers := r.Resource == "*" || r.Resource == action.GetResource().Resource
-	if !resourceCovers {
-		return false
-	}
-
-	return true
+	return resourceCovers(r.Resource, action)
 }
 
 func (r *SimpleProxyReactor) React(action Action) (bool, restclient.ResponseWrapper, error) {
 	return r.Reaction(action)
 }
+
+func resourceCovers(resource string, action Action) bool {
+	if resource == "*" {
+		return true
+	}
+
+	if index := strings.Index(resource, "/"); index != -1 {
+		return resource[:index] == action.GetResource().Resource &&
+			resource[index+1:] == action.GetSubresource()
+	}
+
+	return resource == action.GetResource().Resource
```

Comment:
nit.
it may be slightly better to move this to before subresource match, i.e. the order of matching is 1/ * 2/ resource 3/ subresource.

Reply:
Thanks, Updated.